docs/hello.rst

```diff
@@ -139,7 +138,7 @@ You can also define Taichi **functions** with ``ti.func``, which can be called a
 
```

Comment:
I found that `note`'s L129-138 seems overlaps L127? Could you help to remove that? Thx!

Reply:
> I found that `note`'s L129-138 seems overlaps L127? Could you help to remove that? Thx!

Could you please elaborate on that? I don't really get it 😮

Reply:
Oh, sorry, I mislooked, there's no duplicate at all..